Lib/test/test_typing.py

```diff
@@ -156,6 +156,46 @@ def test_cannot_instantiate(self):
             type(NoReturn)()
 
 
+class NeverTests(BaseTestCase):
```

Comment:
I think we can create something like `_BottomTypeTests` base class and then subclass it with `NoReturnTests` and `NeverTests`.

Reply:
Good idea done. I didn't underscore it by analogy with `NewTypeTests`.